.github/workflows/make-pot.yml

```
@@ -0,0 +1,149 @@
+      - name: Create or Update Pull Request with updated translations
+        if: steps.pot_diff.outputs.pot_changed == 'true'
+        uses: peter-evans/create-pull-request@v6
+        with:
+          add-paths: languages/*
+          commit-message: Update translation files (POT/JSON)
+          title: Update translation files (POT/JSON)
+          body: This PR updates the translation files (POT/JSON) generated by the workflow.
+          branch: update-translations/${{ github.head_ref || github.ref_name }}
+          base: ${{ github.head_ref || github.ref_name }}
+          delete-branch: true
+          update-branch: true
```

actionlint (1.7.7)

input "update-branch" is not defined in action "peter-evans/create-pull-request@v6". available inputs are "add-paths", "assignees", "author", "base", "body", "body-path", "branch", "branch-suffix", "commit-message", "committer", "delete-branch", "draft", "git-token", "labels", "milestone", "path", "push-to-fork", "reviewers", "signoff", "team-reviewers", "title", "token"

(action)